docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,125 @@
+
+### Upgrading Existing System for use as a Server
+
+We recommend Intel ARC GPUs for upgrading existing systems. If Intel ARC GPUs aren't available, we recommend Nvidia GTX 16 / RTX 20 or newer, excluding the GTX 1650.
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, Notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs.
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
+
+#### Integrated Graphics
+
+Integrate graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
+
+#### Resizable BAR
+
+Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.
+
+When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
+
+### System Memory (RAM)
+
+As a general rule of thumb, we recommend at least 8GB for Linux and 16GB for Windows, though significantly smaller amounts may work just fine.
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the hardware acceleration options available is entirely determined by the choice of GPUs.
+
+We recommend Intel graphics over Nvidia graphics because of the quality. We recommend avoiding AMD or Apple Silicon since they produce poor quality output. Encoder Quality: Intel > Nvidia > AMD > Apple
+
+The following is a list of commonly used codecs:
+
+- H.264 (AVC): Blu-Ray, most streaming services
+- H.265 (HEVC): Blu-Ray UHD
+- MPEG-2: DVD
+- VC-1: Blu-Ray
+- VP8, VP9, AV1: streaming services
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
+
+#### Intel Graphics
+
+Please refer to [This Table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
+
+Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.
+
+#### Nvidia Graphics
+
+Please refer to [This Table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
+
+#### AMD Graphics
+
+Please refer to [This Article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
+
+#### Apple Silicon
+
+Supported codecs are listed below:
```

Comment:
Might want to add 10bit HEVC and VP9 capabilities

Reply:
This info isn't published by Apple and the info here was pieced together with info from the internet
There are people who tested VP9 to be supported but I don't think anyone tested 10bit VP9.
I have an M1 Pro device on hand so I can test it out on my system for this particular chip if there are test clips available.

Reply:
I've done some testing and VP9 10bit does hardware decode on my M1 Pro Macbook